test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -664,6 +672,31 @@ Describe "Hard link and symbolic link tests" -Tags "CI", "RequireAdminOnWindows"
             $link.LinkType | Should -BeExactly "SymbolicLink"
             $link.Target | Should -BeExactly $real.ToString()
         }
+
+        It "New-Item can create a directory symbolic link to a directory using a relative path" -Skip:(-Not $IsWindows) {
```

Comment:
Shouldn't this work on on-Windows?

Reply:
Should, but since the file/directory split for symlinks is a Windows-only concept, I don't see much gain from testing for it on Linux/Mac, where it's guaranteed to work anyway and covered by other tests.